frontend/src/scenes/onboarding/sdks/shared-snippets.tsx

```diff
@@ -0,0 +1,27 @@
+import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
+import { Link } from 'lib/lemon-ui/Link'
+
+export function JSInstallSnippet(): JSX.Element {
+    return (
+        <CodeSnippet language={Language.Bash}>
+            {['npm install posthog-js', '# OR', 'yarn add posthog-js', '# OR', 'pnpm add posthog-js'].join('\n')}
```

Comment:
Smart! I like this pattern - I always struggle to write multiline text

Reply:
Not my idea, can't take credit 😄  It's cumbersome with more lines so in other spots I just used backticks..